docs/running.md

```diff
@@ -0,0 +1,209 @@
+
+::: warning
+There's currently no way to use different passwords for different users.
+:::
+
+### Parameter: `mbox`
+
+* Default: `~/mail/dovecot-crlf`
+
+Path to mbox file where to append messages from. See [below](#append-mbox) for how this is used.
+
+## Other Parameters
+
+### Parameter: `box`
+
+* Default: `INBOX`
+
+Mailbox to use for testing.
+
+### Parameter: `clients`
+
+* Default: `10`
+
+Number of simultaneous client connections to use.
+
+### Parameter: `copybox`
+
+* Default: \<none\>
+
+When testing `COPY` command, this specifies the destination mailbox.
+
+### Parameter: `disconnect_quit`
+
+* Default: \<none\>
+
+If a client gets disconnected, quit. Use [logout=0](#parameter-logout) parameter with this. This is useful when debugging problems in the server.
+
+### Parameter: `domains`
+
+* Default: `100`
+
+The upper limit for domain substitution in templates.
+
+### Parameter: `logout`
+
+* Default: TODO
+
+TODO
+
+### Parameter: `master`
+
+* Default: \<none\>
+
+Use master user logins.
+
+### Parameter: `msgs`
+
+* Default: `30`
+
+Try to keep the mailbox size around this many messages.
+
+### Parameter: `no_pipelining`
+
+* Default: `no`
+
+If `yes`, don't send multiple commands at once to server.
+
+### Parameter: `rawlog`
+
+* Default: `no`
+
+Write rawlog.\* files for all connections containing their input and output.
+
+### Parameter: `secs`
+
+* Default: \<none\>
+
+Run ImapTest `n` seconds and then exit.
+
+::: warning
+Setting this to less than the time to run all scripts with `tests=dir` will lead to spurious test failures.
+:::
+
+### Parameter: `seed`
+
+* Default: \<none\>
+
+Seed to use for random generator. Setting this to some specific value makes repeated benchmarks a bit more reliable, because the used commands should be the same.
+
+### Parameter: `users`
+
+* Default: `100`
+
+The upper limit for user substitution in templates.
+
+## Test Selection
+
+### Parameter: State Probabilities
+
+Format: `<state>=<probability>[,<probability2>]`
+
+The state probabilities to use. See [States](/states) for further information.
+
+### Parameter: Checkpoint
+
+* Default: \<none\>
+
+Run a checkpoint every `n` seconds.
+
+### Parameter: `own_msgs`
+
+* Default: \<none\>
+
+Assigns an owner client for each message. Complain if flags for a message is changed by a non-owner session.
+
+### Parameter: `own_flags`
+
+Assigns an owner client for each flag and keyword. Complain if they're changed by another session.
+
+### Parameter: `random`
+
+* Default: `no`
+
+Switch randomly between states.
+
+### Parameter: `no_tracking`
+
+* Default: `no`
+
+Don't track and complain about IMAP state. Makes it use a bit less CPU.
+
+### Parameter: `test`
+
+* Default: \<none\>
+
+Run [scripted tests](/scripted_test) from a given directory instead of doing stress testing.
+
+## TLS/SSL
+
+ImapTest doesn't currently support TLS/SSL directly, but you can use [stunnel](http://www.stunnel.org/) for creating a tunnel for imaps:
```

Comment:
imaptest does support ssl nowadays with ssl or ssl=any-cert parameter. Can remove this stunnel thingy.

Reply:
Done.

Reply:
running.md doesn't exist anymore - it got renamed.  So I think you are looking at the old edit.

Reply:
I made sure to refresh. I think either you forgot to push, or github was buggy. Anyway now the changes are visible.

Reply:
It might have done with the fact that my changes were accidentally amended into the 2nd commit (github actions) instead of the first one... just fixed that.  Anyway, good that it works now.